### ISSUE TITLE
Exodus Loyalty Deck Depletes Completely

### ISSUE THREAD

**Issue by Arcadious** [Jasperhawk/BSG-Moderator-Tools#18]
In BSG V109, a game where the goal for humans is to execute characters, the first few executions were handled by the rules. Boomer's execution was the last execution that the spreadsheet conducted properly, afterwards each execution failed to add a YANAC card to the Exodus loyalty deck, but cards were still dealt out properly. When sleeper arrived, the loyalty deck was short cards, and failed to deal sleeper loyalties out to several players, completely depleting what should be an Exodus loyalty deck.

**Comment by mawkee** [comment]
Do you still have the spreadsheet of that game?

**Comment by Arcadious** [comment]
No, the game finished, and I deleted it. It may have been an issue with the
YANAC deck running out, we did have about a dozen executions, and we were
not using conflicted loyalties.


Ian Andrews Madsen

**Comment by mawkee** [comment]
The YANAC deck runs out eventually. I didn't find any rules regarding reshuffling this deck, and I don't think they apply. I may be wrong.

**Comment by Arcadious** [comment]
I've sent a rules query to FFG. My assumption is that if the "You Are Not a
Cylon" is depleted, that the discards are shuffled into a new deck, just
like any of the other decks.

**Comment by Arcadious** [comment]
Q: When playing with the Exodus expansion, what happens if the "You Are Not
a Cylon" deck is depleted?
A: In the rare event that the "You Are Not a Cylon" deck is depleted,
shuffle all the discarded "You Are Not a Cylon" cards to form a new deck.


Ian Andrews Madsen
